serializable/__init__.py

```diff
@@ -50,15 +58,14 @@
 logger.setLevel(logging.INFO)
 
 
-class _Klass(Protocol):
-    __name__: str
-    __qualname__: str
+class ViewType:
+    """Base of all views."""
+    pass
 
 
 _F = TypeVar("_F", bound=Callable[..., Any])
-_T = TypeVar('_T', bound=_Klass)
-
-ViewType = _Klass
```

Comment:
at runtime, `_Klass`/`ViewType` was just any class. so this stays as is ... 


but the typing was intended to glook tof a `type` of `_Klass`/`ViewType` ...
this is fixed, now